source/math/FloatExtension.ooc

```diff
@@ -120,4 +120,10 @@ extend Float {
 		result := coefficient toString() >> "E" & power toString()
 		result
 	}
+	getTranslationFromRotation: static func (rotation: Float, zDistance: Float) -> Float {
+		sin(rotation) / zDistance
+	}
+	getRotationFromTranslation: static func (translation: Float, zDistance: Float) -> Float {
```

Comment:
`getRotationFromTranslation: static func (translation, zDistance: This) -> This {`